python/lsst/ts/m2gui/controltab/tab_overview.py

```diff
@@ -48,16 +48,10 @@ class TabOverview(TabDefault):
     def __init__(self, title, model):
         super().__init__(title, model)
 
-        # Message signal
-        self._signal_message = None
-
-        # Control signal
-        self._signal_control = None
-
         # Labels
-        self._label_control = QLabel()
-        self._label_control_mode = QLabel()
-        self._label_local_mode = QLabel()
+        self._label_control = create_label()
```

Comment:
You don't gain anything. Calling QLabel directly is perfectly fine, and much easier to read and understand.

Reply:
I will do some modification for `label` in a latter time. This is why I tried to unify the code to use the `create_label()` as a single place. Thanks!